### ISSUE TITLE
cannot connect 

### ISSUE THREAD

**Issue by themass** [rtvi-ai/rtvi-client-android-daily#5]

**Comment by marcus-daily** [comment]
Hi @themass, could you give some more details about the issue you're seeing please?

**Comment by themass** [comment]
![Screenshot_20240919_104302_ai rtvi client basicdemo](https://github.com/user-attachments/assets/86764e84-8b90-4c02-8eb1-22fc79717b01)
![Screenshot_20240919_104345_ai rtvi client basicdemo](https://github.com/user-attachments/assets/1a01a56b-0b5c-4617-beeb-986f85abab9a)

**Comment by marcus-daily** [comment]
Thanks, what do you have the URL set to? It should be set to `https://api.daily.co/v1/bots/start` or another RTVI-compatible backend. Currently it seems to be pointing somewhere which is returning an HTML page.

**Comment by themass** [comment]
i use room url  https://themass.daily.co/voicetest
maybe i am wrong. 
bankend url should use  https://github.com/rtvi-ai/rtvi-infra-examples/tree/main/01-local   ？

**Comment by marcus-daily** [comment]
@themass Yeah the backend URL shouldn't be a room URL. The room is created by the `https://api.daily.co/v1/bots/start` endpoint, or the `http://localhost:7860` endpoint if you deploy locally. The client retrieves the room URL from the RTVI endpoint.

**Comment by themass** [comment]
INFO:     Uvicorn running on http://0.0.0.0:7860 (Press CTRL+C to quit)
INFO:     172.17.51.191:37964 - "POST / HTTP/1.1" 500 Internal Server Error
INFO:     172.17.51.191:34086 - "POST / HTTP/1.1" 500 Internal Server Error
INFO:     172.17.51.191:50512 - "POST / HTTP/1.1" 500 Internal Server Error
INFO:     172.17.51.191:54232 - "POST / HTTP/1.1" 500 Internal Server Error
INFO:     172.17.51.191:54244 - "POST / HTTP/1.1" 500 Internal Server Error


i run https://github.com/rtvi-ai/rtvi-infra-examples/tree/main/01-local, and config bankend url   http://xxxxx:7860/  。 the server throw exception like ""POST / HTTP/1.1" 500 Internal Server Error", there are no other log.
